saltyrtc/server/server.py

```diff
@@ -210,22 +210,22 @@ def handler(self, connection, ws_path):
             self._server.raise_event(Event.disconnected, hex_path, exc.reason)
         except SlotsFullError as exc:
             client.log.notice('Closing because all path slots are full: {}', exc)
-            yield from client.close(code=CloseCode.path_full_error.value)
+            client.close(code=CloseCode.path_full_error.value)
```

Comment:
Hm, here I probably don't quite understand async support in Python. Doesn't "client.close" return a future/promise? Is it still evaluated when removing the "yield from"?

(Btw, "yield from" is the backwards compatible form of "await", right?)

Reply:
> Is it still evaluated when removing the "yield from"?

No. That is essentially launching a coroutine but ignoring the future. But take a look at the subsequent commits. :slightly_smiling_face:

Reply:
Ok, so the coroutines are all implicitly started automatically in the current(?) event loop?

> But take a look at the subsequent commits.

I see :)

Reply:
> Ok, so the coroutines are all implicitly started automatically in the current(?) event loop?

Take a look at the [Task introduction](https://docs.python.org/3/library/asyncio-task.html#coroutines). It also explains the difference between coroutine and task.